### ISSUE TITLE
Child class does not inherit parent scopes

### ISSUE THREAD

**Issue by dja** [hzamani/active_record-acts_as#42]
If I have

``` ruby
class Product
  actable
  scope :no_users, -> { where(user_id: nil) }
end
```

and

``` ruby
class Pen
  acts_as :product
end
```

I can't seem to use Pen.no_users - it throws error: undefined method 'no_users' for Pen

**Comment by BoGs** [comment]
+1

Not only does Pen.no_users not work but if you copy scope to child - the query generated assumes the field is on the child instead of parent. 

I tried using the created_at attribute for my test where Product has the value but Pen inherits it.

**Comment by manuelmeurer** [comment]
Fixed in v2.2.0
